scripts/examples/01-Camera/03-Event-Cameras/02-Genx320/genx320_event_mode_calibration.py

```diff
@@ -0,0 +1,68 @@
+# This work is licensed under the MIT license.
+# Copyright (c) 2013-2025 OpenMV LLC. All rights reserved.
+# https://github.com/openmv/openmv/blob/master/LICENSE
+#
+# This example shows off using the genx320 event camera from Prophesee
+# using event streaming mode and calibrating the camera.
+
+import csi
+import image
+import time
+
+# https://micropython-ulab.readthedocs.io/en/latest/index.html
+from ulab import numpy as np
+
+CALIBRATION_EVENT_COUNT = 10000  # Number of events to collect for calibration.
+CALIBRATION_SIGMA = 0.5  # Standard deviation for hot pixel detection.
+
+# Surface to draw the histogram image on.
+img = image.Image(320, 320, image.GRAYSCALE)
+
+# ndarray to hold events from the camera
+# must be EVT_res events by 6 values
+#
+# 0: event type
+# 1: seconds timestamp
+# 2: milliseconds timestamp
+# 3: microseconds timestamp
+# 4: x coordinate (0-319 for the genx320)
+# 5: y coordinate (0-319 for the genx320)
+events = np.zeros((2048, 6), dtype=np.uint16)
+
+# Initialize the sensor.
+csi0 = csi.CSI(cid=csi.GENX320)
+csi0.reset()
+csi0.ioctl(csi.IOCTL_GENX320_SET_MODE, csi.GENX320_MODE_EVENT,
+           csi.EVT_2048)  # Must be EVT_1024/2048/.../65536
+
+clock = time.clock()
+
+t = time.ticks_ms()
+calibrated = False
+while True:
+    clock.tick()
+
+    # Reads 2048 events from the camera.
+    # Returns the number of valid events (0-2048) or a negative error code.
+    event_count = csi0.ioctl(csi.IOCTL_GENX320_READ_EVENTS, events)
+
+    # Draws the events on the image. If clear=True then the image is cleared
+    # to the default value of "brightness". Event histogram bins have "contrast"
+    # added to them for PIX_ON_EVENT events and subtracted from them for
+    # PIX_OFF_EVENT events clamped between 0 and 255. Pass clear=False to keep
+    # accumulating events in the histogram image.
+    img.draw_event_histogram(events[:event_count], clear=True, brightness=128, contrast=64)
```

Comment:
Should we call this just `draw_events`? And have it flush the buffer?

Reply:
> I didn't want to call it draw_events() as I think that would be like drawing crosses and etc.

The events are not a generic histogram right? If they were we would have called it `draw_histogram`. The function draws something very specific with its format. How about `draw_ec_events` ? Or `draw_genx_events`?

Reply:
Maybe `draw_ec_event_histogram`? I made the ec event stuff to abstract away from the genx320 so everything outside of the camera driver has a more generic name to allow us to support future higher resolution event cameras.

Fine with `draw_ec_events` too, drawing events in a histogram is the only way to really visualize them, the histogram part is redundant.

Reply:
Yeah `draw_ec_events` is fine. What does EC stand for?

Reply:
Actually, I think it should be `draw_histogram` you don't pass the whole events array right? Just a slice which is probably the histogram?

Reply:
Let's just leave it as is, resolving this.